meson_options.txt

```diff
@@ -63,11 +63,11 @@ option('key_screen', type: 'boolean',
   description: 'Enable the key editor screen')
 
 option('lyrics_screen', type: 'boolean',
-  value: false,
+  value: true,
```

Comment:
If you want to suggest enabling the lyrics screen by default, then hiding it inside this PR inside this commit isn't the right way to do it

Reply:
Really sorry, didn't mean to hide it. My version of ncmpc (on arch linux) had lyrics enabled, so I was confused why this wasn't enabled.